libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/features.md

```diff
@@ -5,7 +5,7 @@ description: Features
 slug: radpdfprocessing-formats-and-conversion-pdf-features
 tags: features
 published: True
-position: 1
+position: 2
```

Comment:
This change is related to the deletion of libraries/radpdfprocessing/formats-and-conversion/pdf/pdf.md